portfolio/src/main/webapp/map.html

```diff
@@ -21,6 +22,10 @@
       <div id = "map"></div>
     </div>
     <div id = "results-container"></div>
+    <div id = "chart-container">
+        <div id = "chart-header"><span id = "highlight">What People Prefer To Do</span></div>
+        <div id = "pie-chart"></div>
```

Comment:
here and for map, can you come up with a better name ? you want the name to be related to the intent of the chart; in case you have another pie chart in the future!